app/models/membership_application.rb

```diff
@@ -41,6 +41,8 @@ class MembershipApplication < ApplicationRecord
 
   scope :open, -> { where.not(state: [:accepted, :rejected]) }
 
+  delegate :full_name, to: :user, prefix: true, allow_nil: true
```

Comment:
Why use `allow_nil: true`?

This will simply return `nil` if the `user` for a membership application is `nil`.  However, this is not a convenience but rather indicates a serious logic error somewhere.  Allowing a `nil` return for `user_full_name` would mask this error.

I recommend removing this qualifier.